<ui version="4.0" >
 <class>jewelrySalesClass</class>
 <widget class="QWidget" name="jewelrySalesClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>681</height>
   </rect>
  </property>
  <property name="font" >
   <font>
    <family>微软雅黑</family>
    <pointsize>9</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
    <underline>false</underline>
    <strikeout>false</strikeout>
   </font>
  </property>
  <property name="windowTitle" >
   <string>jewelrySales</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4" >
   <item row="0" column="0" colspan="3" >
    <widget class="QTableWidget" name="tableWidget" >
     <property name="editTriggers" >
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior" >
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text" >
       <string>条码</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>证书编号</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>单位</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>品名</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>类别</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>单价</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>宝石价格</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>工费</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>圈号</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>主石克拉数</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>副石克拉数</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>等级</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>颜色</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>金重</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>总质量</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>数量</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>折扣</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>总金额</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>库存</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QWidget" native="1" name="widget" >
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <property name="leftMargin" >
       <number>1</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>1</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>条码：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leBarcode" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>F1得到光标</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>打印小票</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6" >
        <property name="font" >
         <font>
          <family>SimSun</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600; text-decoration: underline;">L&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QWidget" native="1" name="widget_2" >
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_7" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>18</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>累计金额</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="leAmount" >
        <property name="minimumSize" >
         <size>
          <width>20</width>
          <height>21</height>
         </size>
        </property>
        <property name="font" >
         <font>
          <family>黑体</family>
          <pointsize>20</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="focusPolicy" >
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet" >
         <string notr="true" >background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
font: 75 20pt "黑体";</string>
        </property>
        <property name="text" >
         <string>0.0</string>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QLabel" name="label_10" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>元</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_8" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>18</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>实收金额</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="lePaidUp" >
        <property name="font" >
         <font>
          <family>黑体</family>
          <pointsize>20</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="focusPolicy" >
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="styleSheet" >
         <string notr="true" >font: 75 20pt "黑体";</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QLabel" name="label_11" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>元</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QLabel" name="label_4" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>F2得到光标</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_9" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>18</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>找零金额</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QLabel" name="label_12" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>元</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="leChange" >
        <property name="font" >
         <font>
          <family>黑体</family>
          <pointsize>20</pointsize>
          <weight>9</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="focusPolicy" >
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet" >
         <string notr="true" >background-color: rgb(6, 6, 6);
color: rgb(255, 255, 255);
font: 75 20pt "黑体";</string>
        </property>
        <property name="text" >
         <string>0.0</string>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>label_7</zorder>
     <zorder>leAmount</zorder>
     <zorder>label_10</zorder>
     <zorder>label_8</zorder>
     <zorder>lePaidUp</zorder>
     <zorder>label_11</zorder>
     <zorder>label_4</zorder>
     <zorder>label_9</zorder>
     <zorder>label_12</zorder>
     <zorder>leChange</zorder>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QWidget" native="1" name="widget_3" >
     <layout class="QGridLayout" name="gridLayout_2" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_13" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>18</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>销售人员：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QComboBox" name="cbSaler" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>18</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_15" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>18</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>结算方式：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QComboBox" name="cbPayment" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>18</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <item>
         <property name="text" >
          <string>现金</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>刷卡</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_16" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>18</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>交易折扣：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_17" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>18</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>会员名称：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QLineEdit" name="leVIPCardNo" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>18</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="focusPolicy" >
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QDoubleSpinBox" name="dbTradeDiscounts" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>18</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="minimum" >
         <double>0.010000000000000</double>
        </property>
        <property name="maximum" >
         <double>1.000000000000000</double>
        </property>
        <property name="value" >
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QWidget" native="1" name="widget_4" >
     <layout class="QGridLayout" name="gridLayout_3" >
      <item row="0" column="0" >
       <widget class="QPushButton" name="pushButton" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>18</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>单品折扣:F3</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QPushButton" name="pushButton_3" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>18</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>交易折扣:F4</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QPushButton" name="pushButton_5" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>18</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>删除商品:F5</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0" >
       <widget class="QPushButton" name="pbVIP" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>18</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>结算:Enter</string>
        </property>
       </widget>
      </item>
      <item rowspan="3" row="5" column="0" >
       <widget class="QPushButton" name="pushButton_2" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>18</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>会员:F9</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QPushButton" name="pushButton_10" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>18</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>查询商品:F8</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QPushButton" name="pushButton_7" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>18</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>取消交易:ESC</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QPushButton" name="pushButton_11" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>18</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>修改整单金额</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QPushButton" name="pushButton_12" >
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>18</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>更换操作员</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QPushButton" name="pushButton_15" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>18</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>计算器</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" >
       <widget class="QPushButton" name="pushButton_14" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font" >
         <font>
          <family>微软雅黑</family>
          <pointsize>18</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>退出零售</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>leBarcode</tabstop>
  <tabstop>lePaidUp</tabstop>
  <tabstop>cbSaler</tabstop>
  <tabstop>cbPayment</tabstop>
  <tabstop>leVIPCardNo</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>leChange</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
